alibi/models/pytorch/tests/test_cfrl_models.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import torch
+from alibi.models.pytorch import AE, HeAE
+from alibi.models.pytorch.cfrl_models import (ADULTDecoder, ADULTEncoder,
+                                              MNISTClassifier, MNISTDecoder,
+                                              MNISTEncoder)
+
+
+@pytest.mark.parametrize('output_dim', [10, 20])
+def test_mnist_classifier(output_dim):
+    """ Tests whether the `MNISTClassifier` has a deterministic output when is in the
+     evaluation model and if the output shape matches the expectation. """
+    x = torch.randn(1, 1, 28, 28)
+    model = MNISTClassifier(output_dim=output_dim).cpu()
```

Comment:
Btw are these calls to `cpu` to avoid using any local GPUs when running the test suite?

Reply:
There is no need for the `cpu` when the CI is running since it only has CPU. By default, on my local machine was running on the GPU and the problem was that the tensors were not sent to the GPU. I forgot to delete them after I set the env variable to avoid running on the GPU.